templates/config.json

```diff
@@ -137,6 +137,10 @@
       "id": "program",
       "file": "policies/program.md",
       "name": "Security Program Overview",
+      "instructionsSummary": "Confirm/tailor link to Employee Handbook.",
```

Comment:
I see what you did here, but I'm struggling if it is necessary to have the complexity of a separate `instructionsSummary` field.  It seems a little repetitive/duplicate to the question in the instructions. If a summary is necessary, it can be the first string in the `instructions` array, and it will just render in the UI as the first bullet item.

Reply:
Yeah, of course. I made this a draft to just get some initial feedback from George. We were playing with the idea of having a separate value to capture the summary so that we could use it at a later date, but in my upcoming PR I will be removing instructionsSummary.